kzz7yh-f04172/cod-ve09v2_kzz7yh-f04172.xml

```diff
@@ -78,12 +78,13 @@
    </p>
             <div xml:id="kzz7yh-f04172-Dd1e135">
                <head xml:id="kzz7yh-f04172-Hd1e137">Quaestio 1</head>
+               <head type="question-title">Utrum omnes creature sint ab uno principio</head>
                <p xml:id="kzz7yh-f04172-d1e139">
       Quaesio I. 
       <lb ed="#V" n="97"/>PRimo ostendo quod non omnes 
       <lb ed="#V" n="98"/>creature sint 
-      <lb ed="#V" n="99"/>sab vno principio. Saluator Ioan. 14. Uenit 
-      <lb ed="#V" n="100"/>princepers mundi huius. ibi loquens de diabolo 
+      <lb ed="#V" n="99"/>ab vno principio. Saluator Ioan. 14. Uenit 
+      <lb ed="#V" n="100"/>princepers<!--?--> mundi huius. ibi loquens de diabolo 
```

Comment:
You are correct that this "p" looks like it has a line through it and would normally be be 'per', but I think the word has to be "princeps and I would chalk this up to being a kind of stray mark or very subtle printing error. 

I would keep this as "princeps"

```suggestion
      <lb ed="#V" n="100"/>princeps mundi huius. ibi loquens de diabolo 
```